### PR TITLE
doc: add meeting minutes Aug 8

### LINKED ISSUE #1132: OpenJS Foundation Cross Project Council Meeting 2023-08-08
## Time

**UTC Tue 08-Aug-2023 18:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 08-Aug-2023 11:00 (11:00 AM) |
| US / Mountain | Tue 08-Aug-2023 12:00 (12:00 PM) |
| US / Central  | Tue 08-Aug-2023 13:00 (01:00 PM) |
| US / Eastern  | Tue 08-Aug-2023 14:00 (02:00 PM) |
| EU / Western  | Tue 08-Aug-2023 19:00 (07:00 PM) |
| EU / Central  | Tue 08-Aug-2023 20:00 (08:00 PM) |
| EU / Eastern  | Tue 08-Aug-2023 21:00 (09:00 PM) |
| Moscow        | Tue 08-Aug-2023 21:00 (09:00 PM) |
| Chennai       | Tue 08-Aug-2023 23:30 (11:30 PM) |
| Hangzhou      | Wed 09-Aug-2023 02:00 (02:00 AM) |
| Tokyo         | Wed 09-Aug-2023 03:00 (03:00 AM) |
| Sydney        | Wed 09-Aug-2023 04:00 (04:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Cross%20Project%20Council+Meeting+2023-08-08&iso=20230808T1800
* or https://www.wolframalpha.com/input/?i=06PM+UTC%2C+Aug+08%2C+2023+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1-lglsL8wHpfH_ZM3tgvQIHdVgv4grcHyemAziX7-qsc/edit>

## Agenda

Extracted from **cross-project-council-agenda** labelled issues and pull requests from the **openjs-foundation org** prior to the meeting.

### openjs-foundation/cross-project-council

* Updated OpenJS Foundation Tiered Marketing Offerings [#1128](https://github.com/openjs-foundation/cross-project-council/issues/1128)
* Upcoming CPC Elections [#1126](https://github.com/openjs-foundation/cross-project-council/issues/1126)
* Code of Conduct Plan of Action [#1122](https://github.com/openjs-foundation/cross-project-council/issues/1122)
* add @edsadr as a regular member to CPC [#1121](https://github.com/openjs-foundation/cross-project-council/pull/1121)
* Standardize on Collab Spaces over Working Groups [#1110](https://github.com/openjs-foundation/cross-project-council/issues/1110)
* Non-violent communication / strategic workplace conflict resolution training? [#1106](https://github.com/openjs-foundation/cross-project-council/issues/1106)
* Enable ability to have pseudonimity for EasyCLA. [#1088](https://github.com/openjs-foundation/cross-project-council/issues/1088)
* Requirements for Projects ingesting other existing external projects [#1059](https://github.com/openjs-foundation/cross-project-council/issues/1059)
* JSON Schema initial charter [#1056](https://github.com/openjs-foundation/cross-project-council/issues/1056)
* Simplify CPC Board representation [#1055](https://github.com/openjs-foundation/cross-project-council/pull/1055)
* Open Up Community Board Seat [#1054](https://github.com/openjs-foundation/cross-project-council/issues/1054)
* Should we have a co-chair? [#1043](https://github.com/openjs-foundation/cross-project-council/issues/1043)
* Balancing privacy vs. transparency for the travel fund [#397](https://github.com/openjs-foundation/cross-project-council/issues/397)

## Invited

* OpenJS Foundation Cross Project Council
* OpenJS Foundation Project Maintainers
* OpenJS Foundation Board of Directors

## Notes

The agenda comes from issues labelled with `cross-project-council-agenda` across **all of the repositories in the openjs-foundation org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

link for participants: Zoom link: <https://zoom.us/j/920010234>

* For those who just want to watch: <https://livestream.openjsf.org>

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by joesepi**
closes #1132

### CODE REVIEW

File: meetings/2023/2023-08-08.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# OpenJS Foundation Cross Project Council Meeting 2023-08-08
+
+## Links
+
+* **Recording**:
```
Comment (joesepi) [inline meetings/2023/2023-08-08.md:5]:
```suggestion
* **Recording**: https://www.youtube.com/watch?v=BZnz21hQ83E
```

**Review by mhdawson — APPROVED** [review]
LGTM

File: meetings/2023/2023-08-08.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+## Agenda
+
+### Announcements
+
+* Claudio - nodejs collab summit registration is live
+https://github.com/openjs-foundation/summit/issues/360
+
+### Board Meeting Updates
+
+*Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
```
Comment (VoltrexKeyva) [inline meetings/2023/2023-08-08.md:33]:
```suggestion
* Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
```

File: meetings/2023/2023-08-08.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+### openjs-foundation/cross-project-council
+
+* Upcoming CPC Elections [#1126](https://github.com/openjs-foundation/cross-project-council/issues/1126)
+  * Ben work with Joe on elections, ensure nomination respondes are public (Tobie request)
```
Comment (VoltrexKeyva) [inline meetings/2023/2023-08-08.md:38]:
```suggestion
  * Ben work with Joe on elections, ensure nomination responses are public (Tobie's request)
```

File: meetings/2023/2023-08-08.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+* Code of Conduct Plan of Action [#1122](https://github.com/openjs-foundation/cross-project-council/issues/1122)
+  * progress is happening; take a look at open PRs; any help is appreciated
+
+* add @edsadr as a regular member to CPC [#1121](https://github.com/openjs-foundation/cross-project-council/pull/1121)
+  * Welcome to Adrian; as end user director
```
Comment (VoltrexKeyva) [inline meetings/2023/2023-08-08.md:44]:
```suggestion
  * Welcome to Adrian; as end-user director
```

File: meetings/2023/2023-08-08.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+* Standardize on Collab Spaces over Working Groups [#1110](https://github.com/openjs-foundation/cross-project-council/issues/1110)
+  * Joe working on it
+  * Tobie, double checking on discussion with standard working group
+  * Revisit code of conduct working group, decision made to close since this is now part of CPC
+
+* Non-violent communication / strategic workplace conflict resolution training? [#1106](https://github.com/openjs-foundation/cross-project-council/issues/1106)
+  * Robin - next step is to go back to LF HR and see what it would take to put together a training for us, will also check on previous training (that Tracy had helped organize)
+
+* Enable ability to have pseudonimity for EasyCLA. [#1088](https://github.com/openjs-foundation/cross-project-council/issues/1088)
```
Comment (VoltrexKeyva) [inline meetings/2023/2023-08-08.md:55]:
```suggestion
* Enable the ability to have pseudonymity for EasyCLA. [#1088](https://github.com/openjs-foundation/cross-project-council/issues/1088)
```

Comment (tobie) [reply meetings/2023/2023-08-08.md:55]: That's the actual title of the issue, so let's not fix it here.

File: meetings/2023/2023-08-08.md (hunk 4 above)
Comment (tobie) [reply meetings/2023/2023-08-08.md:44]: The actual title is End User Director (without the hyphen), so let's leave it as such.

**Review by tobie — APPROVED** [review]

**Review by VoltrexKeyva — APPROVED** [review]

**Pull request merged**